docs/user_documentation/ref-emx.md

```diff
@@ -103,8 +103,7 @@ Packages:
 For all technical names in the EMX format, the following rules apply:
 - No special characters, except for; '_' and '#', only letters, numbers are allowed.
 - No names starting with digits. 
-- Maximum length for names is 30 chars.
-- Keywords used by programming languages (e.g. java, javascript, postgresql) are not allowed.
+- The keywords: "login", "logout", "csv", "base", "exist", "meta", are not allowed.
```

Comment:
Application will break for the additional keywords: version, i18n, copy

Reply:
created: https://github.com/molgenis/molgenis/issues/7011